projects/step-core/src/lib/client/augmented/services/augmented-bookmarks.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { StepDataSource } from '../../table/step-table-client.module';
+import { BookmarkService } from '../../../services/bookmark.service';
+import { Bookmark } from '../../generated/models/Bookmark';
+
+@Injectable({ providedIn: 'root' })
+export class AugmentedBookmarksService extends BookmarkService {
```

Comment:
I don't understand why we have bookmarkservice and augmentedbookmarkservice. Couldn't the createDataSource function not be part of the bookmarkservice?